<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendMessageFrame</class>
 <widget class="QFrame" name="SendMessageFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px; background-color: #212529; color: #aaa</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Lato</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #ddd; 
font-size: 20px;
font-family: Lato;</string>
       </property>
       <property name="text">
        <string>SEND MESSAGE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_sendButton">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>171</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#m_sendButton {color: orange; background-color: #1; border: 1px solid orange; font-size: 16px;}

QPushButton#m_sendButton:hover {color: orange; background-color: #1; border: 2px solid orange; font-size: 16px;}</string>
       </property>
       <property name="text">
        <string>SEND MESSAGE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_backButton">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>171</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#m_backButton {color: #ccc; border: 1px solid #343a40; font-size: 16px; background-color: #21259;}

QPushButton#m_backButton:hover {color: #ddd; border: 2px solid #444; font-size: 16px; background-color: #21259;}</string>
       </property>
       <property name="text">
        <string>BACK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="m_messageAddressScrollArea">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1240</width>
        <height>233</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="m_addressesLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>139</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Lato</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Encrypted message:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="m_messageTextEdit">
     <property name="font">
      <font>
       <family>Lato</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #333; color: #ddd;</string>
     </property>
     <property name="html">
      <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lato'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.Helvetica Neue DeskInterface'; font-size:13pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="m_addReplyToCheck">
       <property name="font">
        <font>
         <family>Lato</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 3px; border: 0px;</string>
       </property>
       <property name="text">
        <string>Add &quot;Reply To&quot;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QCheckBox" name="m_ttlCheck">
       <property name="font">
        <font>
         <family>Lato</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Do not include message in blockchain</string>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 3px; border: 0px;</string>
       </property>
       <property name="text">
        <string>Set self destruct time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="m_ttlSlider">
       <property name="singleStep">
        <number>300</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_ttlLabel">
       <property name="font">
        <font>
         <family>Lato</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 3px; border: 0px;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Lato</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 3px; border: 0px;</string>
       </property>
       <property name="text">
        <string>Fee:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>298</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="m_feeSpin">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Lato</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string>padding: 3px; border: 0px; background-color: #777;</string>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>0.000010000000000</double>
       </property>
       <property name="maximum">
        <double>100000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000010000000000</double>
       </property>
       <property name="value">
        <double>0.000010000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_sendButton</sender>
   <signal>clicked()</signal>
   <receiver>SendMessageFrame</receiver>
   <slot>sendClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_messageTextEdit</sender>
   <signal>textChanged()</signal>
   <receiver>SendMessageFrame</receiver>
   <slot>messageTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_ttlSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SendMessageFrame</receiver>
   <slot>ttlValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_ttlCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_ttlSlider</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_ttlCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_ttlLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_ttlCheck</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SendMessageFrame</receiver>
   <slot>ttlCheckStateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_backButton</sender>
   <signal>clicked()</signal>
   <receiver>SendMessageFrame</receiver>
   <slot>backClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1198</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addressBookClicked()</slot>
  <slot>pasteClicked()</slot>
  <slot>sendClicked()</slot>
  <slot>backClicked()</slot>
  <slot>mixinValueChanged(int)</slot>
  <slot>addressEdited(QString)</slot>
  <slot>messageTextChanged()</slot>
  <slot>addRecipientClicked()</slot>
  <slot>ttlCheckStateChanged(int)</slot>
  <slot>ttlValueChanged(int)</slot>
 </slots>
</ui>
